aspnetcore-webapi/BuildQueryString/BuildQueryString/HttpClientWrapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace BuildQueryString
+{
+    public class HttpClientWrapper : IHttpClientWrapper
+    {
+        private static readonly HttpClient _httpClient = new ();
```

Comment:
Remove the space between 'new' and '()'.

Reply:
Updated

Reply:
Can you please confirm if you have committed this change?

Reply:
It should reflect now.